hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/deployment/JetClassLoader.java

```diff
@@ -76,28 +66,6 @@ public JetClassLoader(@Nonnull NodeEngine nodeEngine,
         return defineClass(name, classBytes, 0, classBytes.length);
     }
 
-    @Override
-    protected URL findResource(String name) {
```

Comment:
Why was it necessary to remove this method? What if there's a jar on classpath and it wants to read some xml file on its classpath?